### DISCUSSION TITLE
Unable to join GitHub Education because my academic status won’t get approved.

### DISCUSSION THREAD

**Discussion by realgorgan** [community/community#126511] [GitHub Education]
### Hi everyone,

Hello! I’ve been trying to sign up for GitHub Education and I keep getting declined. I have valid student academic identification however GitHub won’t approve it.

**Comment by realgorgan** [comment]
![IMG_0286](https://github.com/community/community/assets/70828143/25c4a64c-3777-4fbd-a7bb-0baa06e9a85a)

**Comment by pkprajapati7402** [comment]
Hi dude, I also got the same type of message, while getting the Student Developer Pack. This can be fixed via

1. Make sure that you only upload the Clear image of your College ID card in full size. which is representing clear name of your college and Enrollment Number/Roll Number of you. also do not involve any distraction in the image like(blur image, strap of ID Card, Background borders of image).
2. use croping tools to only have the picture of id card only.
3. 
![image](https://github.com/community/community/assets/167101466/626c2e8b-e67d-433b-8bff-344d84e24417)

these is how you should upload your id card in specified size.
In my case also the clarity was the main issue but I solved it by uploading clear image of ID

**Comment by mishmanners** [comment]
Thanks for submitting this. I'll pass it onto the relevant team 👍
